Flow.Launcher.Infrastructure/UserSettings/Settings.cs

```diff
@@ -220,4 +223,44 @@ public enum ColorSchemes
         Light,
         Dark
     }
+
+    public struct ShortCutModel
```

Comment:
can we have the naming consistent with custom hotkey please i.e. CustomQueryShortcut/CustomShortcut -> CustomQueryShortcutModel/CustomShortcutModel.

Reply:
Sure, just feel like it is a bit redundant.

Reply:
Well naming it shortcut is too broad, since they both relate to setting custom queries, i think they fall into the same category and better to maintain consistency in the codebase.

Reply:
@taooceros